tests/unit/conftest.py

```diff
@@ -202,7 +200,11 @@ def pyramid_config(pyramid_request, lti_v11_params):
         # Align request.identity with request.lti_user
         config.testing_securitypolicy(
             userid=lti_v11_params["user_id"],
-            identity=Identity(lti_v11_params["user_id"], permissions=[]),
+            identity=Identity(
+                lti_v11_params["user_id"],
+                permissions=[Permissions.LTI_LAUNCH_ASSIGNMENT],
+            ),
+            permissive=False,
```

Comment:
I reckon permissive=True is a bad default, see:

https://github.com/hypothesis/lms/pull/6377/commits/6b1fd3c2c40600806ca3d1d6a0b2875f3793c36b